src/components/FinancialInfo.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { Typography, Button, Divider } from '@material-ui/core';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+import TextField from './TextField';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      color: theme.palette.text.primary,
+    },
+    content: {
+      marginTop: '10px',
+      border: `1px solid ${theme.palette.divider}`,
+      borderRadius: '6px',
+      padding: '20px',
+    },
+    itemWrapper: {
+      display: 'flex',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+    },
+    right: {
+      textAlign: 'right',
+    },
+    bolded: {
+      fontWeight: 700,
+    },
+    divider: {
+      margin: '10px 0px',
+    },
+    balanceText: {
+      fontWeight: 700,
+      color: theme.palette.primary.main,
+    },
+    fieldContainer: {
+      marginTop: '10px',
+    },
+  }),
+);
+
+interface CardProps {
+  bankName: string;
+  accountNumber: number;
+  accountName: string;
+  balance: number;
+}
+
+export default function FilledCard(props: CardProps) {
```

Comment:
Usually react components are named after their file name (or vice versa). Lets think of a better name for this component (and the props usually share the same name as the component appended with `Props`)